hack/tools/go.mod

```diff
@@ -17,9 +17,12 @@ require (
 	gotest.tools/gotestsum v1.12.0
 	honnef.co/go/tools v0.6.0
 	k8s.io/publishing-bot v0.5.0
+	sigs.k8s.io/kube-api-linter v0.0.0-20250501211755-2c83ed303cde
 	sigs.k8s.io/logtools v0.8.1
 )
 
+replace sigs.k8s.io/kube-api-linter => github.com/JoelSpeed/kube-api-linter v0.0.0-20250502081000-7a11964aec14
```

Comment:
Need to add the Go plugin version to the repo, have a WIP branch, will get a PR up so we can remove this